shared/slotutil/slotticker.go

```diff
@@ -51,9 +51,9 @@ func GetSlotTicker(genesisTime time.Time, secondsPerSlot uint64) *SlotTicker {
 	return ticker
 }
 
-// GetSlotTickerWithOffset is a constructor for SlotTicker that allows a offset of time from genesis,
+// NewWithOffset starts and returns a SlotTicker instance that allows a offset of time from genesis,
 // entering a offset greater than secondsPerSlot is not allowed.
-func GetSlotTickerWithOffset(genesisTime time.Time, offset time.Duration, secondsPerSlot uint64) *SlotTicker {
+func NewWithOffset(genesisTime time.Time, offset time.Duration, secondsPerSlot uint64) *SlotTicker {
```

Comment:
We don't know what is that thing with offset. I would prefer `NewSlotTickerWithOffset`.